README.md

```diff
@@ -1,6 +0,0 @@
-# Populating an excel sheet with Gspread library
```

Comment:
Your readme is a little incomplete. Include details like the author of the project, the tools you used, the source of the data used, and other details you feel are important.